### PR TITLE
Issue 48 feat/UI create nav bar

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

**Comment by Veryyapeee** [comment]
Why I can't navigate to any route and i'm redirected to 404?

File: src/App.jsx [hunk 1]
```diff
@@ -26,15 +26,7 @@ export const App = () => {
```
Comment (Veryyapeee) [inline src/App.jsx:26]: remove every change in this file

File: src/components/organisms/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        []
+    );
+
+    return (
+        <div className='nav__bar'>
+            <Logo className='nav__bar__logo' />
+            <div className='nav__bar__section'>
+                <UserInfo onClick={handleOnImgClick} />
+                <Navigation isOpen={isNavigationOpen} />
+                <ImageComponent src={COOKIE} alt='' size='large' onClick />
```
Comment (Veryyapeee) [inline src/components/organisms/NavBar/NavBar.jsx:23]: empty onClick function?

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/organisms/NavBar/NavBar.jsx (hunk 2 above)
Comment (Mariusz-web) [reply src/components/organisms/NavBar/NavBar.jsx:23]: When I was giving console.log, github was screaming. In next task I add function to onClick

File: src/App.jsx (hunk 1 above)
Comment (Mariusz-web) [reply src/App.jsx:26]: Ok, i remove change

Comment (Mariusz-web) [reply src/App.jsx]: done

**Comment by Veryyapeee** [comment]
Where is this navbar? xd image

File: src/components/organisms/Header/Header.jsx [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
     return (
-        <header>
-            <Link to={routes.Home}>Home</Link>
-            <Link to={`${routes.Feed}${routes.Profile}/${profileID}`}>
-                Profile
-            </Link>
-            <Link to={`${routes.Feed}${routes.Chat}`}>Chat</Link>
-            <Link to={`${routes.Feed}${routes.EditProfile}`}>Edit Profile</Link>
-        </header>
+        <div className='nav__bar'>
```
Comment (Veryyapeee) [inline src/components/organisms/Header/Header.jsx:18]: That should be header tag

**Comment by Mariusz-web** [comment]
Corrected

**Comment by T0ffik** [comment]
Please delete this underlines 
image

File: src/components/organisms/Navigation/Navigation.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            className={`navigation ${
+                isOpen ? 'dropdown__open' : 'dropdown__close'
+            }`}
+        >
+            <Text customClass='navigation__username'>{userName}</Text>
+            {NAV_ITEMS.map((navItem) => (
+                <NavLink to={navItem.path} key={navItem.src}>
+                    <InformationRow
+                        iconName={navItem.src}
+                        onClick={navItem.onClick}
```
Comment (Veryyapeee) [inline src/components/organisms/Navigation/Navigation.jsx:36]: NavItem dont even have onClick, also clicking on elements in header im redirected to 404

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/organisms/Header/Header.jsx [hunk 5]
```diff
@@ -1,18 +1,27 @@
...
+            <Logo className='nav__bar__logo' />
```
Comment (T0ffik) [inline src/components/organisms/Header/Header.jsx:19]: Logo should redirect to main screen(/feed)

File: src/components/organisms/Header/Header.jsx (hunk 3 above)
Comment (Mariusz-web) [reply src/components/organisms/Header/Header.jsx]: done

File: src/components/organisms/Navigation/Navigation.jsx (hunk 4 above)
Comment (Mariusz-web) [reply src/components/organisms/Navigation/Navigation.jsx:40]: done

**Comment by Mariusz-web** [comment]
@T0ffik, I deleted it

File: src/components/organisms/Header/Header.scss [hunk 6]
```diff
@@ -1,9 +1,47 @@
...
+    height: auto;
     display: flex;
-    flex-direction: column;
-    overflow: hidden;
+    flex-direction: row;
+    align-items: center;
+    background-color: $secondary-backgroundColor;
+    padding: 1rem 0;
+    justify-content: space-between;
+    position: relative;
```
Comment (T0ffik) [inline src/components/organisms/Header/Header.scss:13]: Same here

File: src/components/organisms/Header/Header.scss [hunk 7]
```diff
@@ -1,9 +1,47 @@
 @import '../../../styles/variables.scss';
 @import '../../../styles/mixins.scss';
 
-header {
-    width: 15%;
+.nav__bar {
+    width: 100%;
+    height: auto;
```
Comment (T0ffik) [inline src/components/organisms/Header/Header.scss:6]: These style look like change nothing.

**Review by Veryyapeee — APPROVED** [review]
I think it's looking pretty good, just last thing (i'm approving anyway), add cursor: pointer for image when the header is on top (mobile mode)

**Review by T0ffik — CHANGES_REQUESTED** [review]

**Review by T0ffik — APPROVED** [review]

**Pull request merged**